Dockerfile

```diff
@@ -2,42 +2,41 @@ FROM ubuntu:16.04
 
 ENV DEBIAN_FRONTEND=noninteractive
 
+ENV PORT 80
+
+EXPOSE 80
+
 RUN apt-get update -y && \
     apt-get install -y software-properties-common && \
-    apt-add-repository -y ppa:brightbox/ruby-ng && \
-    apt-get update -y
-
-RUN apt-get install -y \
-      build-essential \
-      ruby2.5 ruby2.5-dev \
-      postgresql-client \
-      libpq-dev \
-      unattended-upgrades \
-      ldap-utils \
-      git \
-      curl \
-      update-notifier-common \
-      tzdata
+    apt-add-repository -y ppa:brightbox/ruby-ng
+
+RUN apt-get update -y && \
+    apt-get install -y build-essential \
+                       curl \
+                       git \
+                       libpq-dev \
+                       ldap-utils \
+                       postgresql-client \
+                       ruby2.5 ruby2.5-dev \
+                       tzdata \
+                       unattended-upgrades \
+                       update-notifier-common
 
 RUN gem install -N -v 1.16.1 bundler
 
 RUN mkdir -p /opt/conjur-server
 
 WORKDIR /opt/conjur-server
 
-ADD Gemfile      .
-ADD Gemfile.lock .
+COPY Gemfile \
+     Gemfile.lock ./
 
 RUN bundle --without test development
 
-ADD . .
+COPY . .
```

Comment:
Are all of the changes to this file just syntax? The only one that looks like it might not be is this line. Is this part of the fix?

Reply:
They're not just syntax, they're a lot are reordering to reduce/reuse cache layers. This particular change though is due to the fact that `ADD` is generally used to put something in the image from a compredded file or the web vs the `COPY` which is an explicit local FS copy.